Documentation/security/aws.rst

```diff
@@ -144,7 +144,7 @@ status of ``coredns`` you can run the following command:
 
 .. code-block:: shell-session
 
-    $ kubectl get deployment kube-dns -n kube-system
+    $ kubectl get deployment coredns -n kube-system
```

Comment:
Seems like although coredns has been the default for a while, kube-dns is still maintained and users might still be using it?

This seems like it should work for both:
```suggestion
    $ kubectl get deployment -l k8s-app=kube-dns -n kube-system
```